src/backend/commands/vacuum.c

```diff
@@ -2779,14 +2788,16 @@ vacuum_appendonly_index(Relation indexRelation,
 						int elevel)
 {
 	IndexBulkDeleteResult *stats;
-	IndexVacuumInfo ivinfo;
+	IndexVacuumInfo ivinfo = {0};
```

Comment:
postgres likes use MemSet for such case

Reply:
Indeed, MemSet is a typical macro to fill with zeroes any (heap or stack) structs in the Postgres code base.
But, `= {0}` initializer is already widely used in the Greenplum codebase.
I chose the kind that was [used](https://github.com/greenplum-db/gpdb/blob/6361274560592b4e6026e5c951527651d6645140/src/backend/commands/vacuum_ao.c#L520) for upstream master branch and had lost during porting to 6X.
An interesting article with C standard references about this topic: https://www.ex-parrot.com/~chris/random/initialise.html